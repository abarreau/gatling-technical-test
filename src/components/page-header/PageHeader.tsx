import { FC, ReactNode } from 'react';
import styles from './PageHeader.module.css';

/**
 * A page header is meant to be displayed on top of each page for app names, nav buttons, etc
 */
export const PageHeader: FC<{ children?: ReactNode }> = ({ children }) => {
    return (<div className={styles.mainHeader}>{ children }</div>);
};