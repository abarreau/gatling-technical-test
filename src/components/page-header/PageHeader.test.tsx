import renderer from 'react-test-renderer';
import { PageHeader } from './PageHeader';

describe('PageHeader', () => {
    it('should display component', () => {
        const component = renderer.create(
            <PageHeader>My header</PageHeader>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
