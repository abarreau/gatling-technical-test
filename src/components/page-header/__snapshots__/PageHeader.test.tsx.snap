// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader should display component 1`] = `
<div
  className="mainHeader"
>
  My header
</div>
`;
